prow/moonraker/client.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package moonraker
```

Comment:
Let's do this in a follow-up. I assume that you're thinking of `prow/moonraker/client/client.go` for the new path (so that other components that use the client will import `k8s.io/test-infra/prow/moonraker/client` instead of `k8s.io/test-infra/prow/moonraker`)?